src/components/InstancesList/InstancesList.js

```diff
@@ -93,6 +104,58 @@ class InstancesView extends React.Component {
     <FilterNavigation segment={this.props.segment} />
   );
 
+  generateInTransinItemReport = async () => {
+    const {
+      reset,
+      GET,
+    } = this.props.parentMutator.itemsInTransitReport;
+
+    try {
+      reset();
+      const items = await GET();
+
+      if (items.length > 0) {
```

Comment:
`isEmpty` is more descriptive so I would vote for that.
```suggestion
      if (!isEmpty(items)) {
```